### PR TITLE
Add gas oracle strategies to chain-idiosyncracies.md

### LINKED ISSUE #906: Add comments about gas price strategies in `chain-idiosyncrasies.md`

### PR DESCRIPTION
**Pull request by vponline**
Closes #906.

### CODE REVIEW

File: docs/airnode/v0.8/reference/chain-idiosyncrasies.md [hunk 1]
```diff
@@ -26,10 +26,11 @@ and for the corresponding reference section
 
 ### Avalanche
 
-Gas fees on Avalanche mainnet are often underestimated when using `legacy` as a
-`txType`. The error manifests as the max fee per gas being set to less than the
-block base fee, resulting in unfulfilled requests. A solution to this is to set
-`gasPriceMultiplier` to slightly greater than `1` e.g. `1.1`.
+Gas prices on Avalanche mainnet are often underestimated when using getting
+`type 0` estimates (e.g. for the `providerRecommendedGasPrice` strategy). The
+error manifests as the `maxFeePerGas` being set to less than the block
```
Comment (dcroote) [inline docs/airnode/v0.8/reference/chain-idiosyncrasies.md]: Deletes "using". Prettier adjusts the next two lines after that.

```suggestion
Gas prices on Avalanche mainnet are often underestimated when getting `type 0`
estimates (e.g. for the `providerRecommendedGasPrice` strategy). The error
manifests as the `maxFeePerGas` being set to less than the block
```

**Review by dcroote — APPROVED** [review]
One minor typo, but otherwise looks good.

**Comment by wkande** [comment]
@aquarat How did we get the cloud flare previews. Is cloudflare link to the repo and just knows how to insert them into PRs. Did we configure them some other way?

**Review by wkande — APPROVED** [review]
Look good thanks

**Pull request merged**